src/middlewares/articles.js

```diff
@@ -22,14 +21,10 @@ export default {
  */
   checkArticle: async (request, response, next) => {
```

Comment:
Please properly align the JSDoc for this method. Could you please also explain what this function is doing because it seems hard to understand.

Reply:
> Please properly align the JSDoc for this method. Could you please also explain what this function is doing because it seems hard to understand.

It checks if the followupId is a valid author id and also checks if any of the tags provided is invalid or is not concurrent with the predefined categories in our database.
It just checks to ensure that followupId and tags are valid.